ffi/diplomat/js/examples/wasm-demo/src/ts/date-time.ts

```diff
@@ -111,7 +111,7 @@ export class DateTimeDemo {
                 this.#displayFn("");
             }
         } catch (e) {
-            this.#displayFn(`Error: ${e.error_value}`);
+            this.#displayFn(`Error: ${e}`);
```

Comment:
We should probably change this back

Reply:
I don't think we should, this makes it easier to debug, same reason why we turned off minification.